doc/api.apib

```diff
@@ -174,11 +183,342 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
         {
-            "docker_image": "busybox:latest"
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+        }
+
++ Response 200 (application/json)
+
+        {
+            "enabled": null ,
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+            "next_natural_trigger": null
+        }
+
+# Group Resources
+
+## Resources [/v2/resources]
+
+### Get Resources [GET]
+
++ Response 200 (application/json)
+
+        {
+            "resources": [{
+                "concurrency": 20,
+                "id": "luigi-canary-resource"
+            }]
+        }
+
+### Create Resource [POST]
+
++ Request (application/json)
+
+        {
+            "concurrency": "20",
+            "id": "luigi-canary-resource"
+        }
+
++ Response 200 (application/json)
+
+        {
+            "concurrency": 20,
+            "id": "luigi-canary-resource"
+        }
+
+## Resource [/v2/resources/{resource}]
+
++ Parameters
+    + resource: `luigi-canary-resource` (string) - Resource ID
+
+### Get Resource [GET]
+
++ Response 200 (application/json)
+
+        {
+            "resources": [{
+                "concurrency": 20,
+                "id": "luigi-canary-resource"
+            }]
+        }
+
+### Delete Resource [DELETE]
+
++ Response 204
+
+### Modify Resource [PUT]
+
++ Request (application/json)
+
+        {
+            "concurrency": "30",
+            "id": "luigi-canary-resource"
+        }
+
++ Response 200 (application/json)
+
+        {
+            "concurrency": 30,
+            "id": "luigi-canary-resource"
+        }
+
+# Group Backfills
+
+## Backfills [/v2/backfills{?component,workflow,status,showAll}]
+
+### Get Backfills [GET]
+
++ Parameters
+    + component: `styx-canary` (optional, string) - Workflow Component to filter with
+    + workflow: `LuigiCanary` (optional, string) - Workflow ID to filter with
+    + status: `false` (optional, boolean) - If to return the status list per backfill
+      + Default: false
+    + showAll: `false` (optional, boolean) - If to return also halted backfills and backfills whose instances have been all triggered
+      + Default: false
+
++ Response 200 (application/json)
+
+        {
+            "backfills": [{
+                "backfill": {
+                    "all_triggered": false,
+                    "concurrency": 10,
+                    "end": "2017-02-00T00:00:00Z",
+                    "halted": false,
+                    "id": "backfill-1489054446085-52684",
+                    "next_trigger": "2017-01-01T15:00:00Z",
+                    "schedule": "hours",
+                    "start": "2017-01-01T00:00:00Z",
+                    "workflow_id": {
+                        "component_id": "styx-canary",
+                        "id": "LuigiCanary"
+                    }
+                },
+                "statuses": null
```

Comment:
it is good enough to copy https://github.com/spotify/styx/pull/115/files/d2d0d6f8edaac0001ae91eff69f8dd2759685881#diff-cfe769770442367be801c65d5428a63cR380 here